### DISCUSSION TITLE
How to sort Page Tree by something other than alphabetical name of pages

### DISCUSSION THREAD

**Discussion by ryangriggs** [requarks/wiki#7487] [Help / Questions] [answered]
Hi, I am using wiki.js to create a documentation server for an open source project.  In the documentation, it doesn't necessarily make sense to sort all pages by alphabetical order. Is it possible to sort the pages in Tree View by some other property, or drag and drop the pages into the desired order?  I realize I can use a custom menu, but then I have to add every page to the menu manually. I was hoping for a button that can be ticked which sorts by page ID, modification date, custom property value, etc.

Thank you for any pointers.

**Comment by NGPixel** [comment, marked as answer]
Not at the moment.
